scripts/devops_tasks/test_run_samples.py

```diff
@@ -134,8 +134,6 @@
     "azure-ai-formrecognizer": [
         "sample_manage_custom_models.py",
         "sample_manage_custom_models_async.py",
-        "sample_copy_model.py",
-        "sample_copy_model_async.py",
```

Comment:
Checked async sample and it doesnt have this issue...must have been a copy paste error :/